packages/vue/src/components/result/ResultList.jsx

```diff
@@ -0,0 +1,113 @@
+import { RLConnected } from './ReactiveList.jsx';
+import Title from '../../styles/Title';
+import VueTypes from 'vue-types';
+import ListItem, { container, Image } from '../../styles/ListItem';
+import types from '../../utils/vueTypes.js';
+import { helper } from '@appbaseio/reactivecore';
+
+const { getClassName } = helper;
+
+const ResultList = {
+	name: 'ResultList',
+	props: {
+		currentPage: VueTypes.number.def(0),
+		includeFields: types.includeFields.def(['*']),
+		// component props
+		className: types.string,
+		componentId: types.stringRequired,
+		dataField: types.stringRequired,
+		defaultQuery: types.func,
+		excludeFields: types.excludeFields.def([]),
+		innerClass: types.style,
+		listClass: VueTypes.string.def(''),
+		loader: types.title,
+		onAllData: types.func,
+		onData: types.func,
+		onResultStats: types.func,
+		onNoResults: VueTypes.string.def('No Results found.'),
+		pages: VueTypes.number.def(5),
+		pagination: VueTypes.bool.def(false),
+		paginationAt: types.paginationAt.def('bottom'),
+		react: types.react,
+		showResultStats: VueTypes.bool.def(true),
+		size: VueTypes.number.def(10),
+		sortBy: types.sortBy,
+		sortOptions: types.sortOptions,
+		stream: types.bool,
+		URLParams: VueTypes.bool.def(false),
+		target: VueTypes.string.def('_blank')
+	},
+	render() {
+		const { onData, ...props } = this.$props;
+		return (
+			<RLConnected
+				{...{
+					props: {
```

Comment:
What's the need to write it like this, is simple props passing not working here?
```
<RLConnected
   listClass={container}
  {...on: {  data: this.renderAsList }}
/>
```
should work

Reply:
I tried this it wasn't working , will confirm once otherwise we can keep it like this as stated in the plug-in docs

Reply:
ok

Reply:
I tried this it wasn't working , will confirm once otherwise we can keep it like this as stated in the plug-in docs

Reply:
We also need to pass other props using destructuring so we would need props eventually for that as well